docs/en/index.md

```diff
@@ -7,14 +7,22 @@ nav_order: 15
 lang: en
 ---
 
-Welcome to the documentation center for User Management APIs from Adobe.
+Welcome to the documentation center for User Management APIs from Adobe.  
+
+News:  
 
 <div class="isa_info">
 <p>On April 2nd, 2019, Creative Cloud product names will remove the “CC”. For example, “Photoshop CC” will be renamed “Photoshop”.</p>
-<p>As a result, any applications directly accessing the User Management API which include logic <strong>dependent on the product name</strong> will need to be updated. If you have not included the product name in the code, then this will not impact your connection to the User Management API. If you use the User Sync Tool, you should see no impact.</p>
+<p>As a result, any application directly accessing the User Management API which include logic <strong>dependent on the product name</strong> will need to be updated. If you have not included the product name in the code, then this will not impact your connection to the User Management API. If you use the User Sync Tool, you should see no impact.</p>
 <p>Note: the term “CC” included in the name of a Product Profile <em>will not change</em>. If you rely on the name of the “product admin group” (e.g., <code>_admin_&lt;product name&gt;</code>) you may be impacted and have to update your scripts.</p>
 <p>As a best practice, it is recommended to avoid any logic that expects fixed product names.</p>
 </div>
+<hr class="api-ref-rule">
```

Comment:
Would people expect to see the latest news first?

Reply:
You're right! I'll fix to have thhe latest news on top.